projects/bootloader/test/test_bootloader_crc32.c

```diff
@@ -0,0 +1,55 @@
+#include <stdlib.h>
+
+#include "bootloader_crc32.h"
+#include "bootloader_mcu.h"
+#include "config.h"
+#include "crc32.h"
+#include "flash.h"
+#include "interrupt.h"
+#include "log.h"
+#include "persist.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+uint32_t s_crc32_codes_size = 54;
+
+// intialize memory with random numbers for testing
+void intialize_memory() {
```

Comment:
Nit: the parameter list should be `(void)`. Also, typo: `intialize -> initialize`.

Reply:
done